terraform/basic_components/main.tf

```diff
@@ -0,0 +1,51 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Create a basic component that can be shared across so we don't have to change every terraform that using the same resources 
* [ec2 windows vpc](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/ec2/win/vpc.tf)
* [ec2 mac vpc](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/ec2/mac/vpc.tf)